apps/web/components/BundleOrderItems/BundleOrderItems.vue

```diff
@@ -21,25 +20,41 @@
             <span class="underline h-auto">{{ productBundleGetters.getBundleItemName(item) }}</span>
           </p>
         </SfLink>
-        <p class="font-medium text-sm" v-else>
-          {{ productBundleGetters.getBundleItemQuantity(item) }}x
-          <span class="h-auto">{{ productBundleGetters.getBundleItemName(item) }}</span>
-        </p>
         <div
           class="h-auto line-clamp-3 mt-1 font-normal typography-text-sm"
           v-html="productBundleGetters.getBundleItemDescription(item)"
         ></div>
       </div>
+      <div v-else>
+        <p class="font-medium text-sm">
+          {{ productBundleGetters.getBundleItemQuantity(item) }}x
+          <span class="h-auto">{{ t('products') }}</span>
+        </p>
+        <div
+          class="h-auto mt-1 font-normal typography-text-sm"
+          v-html="t('productAttributes.productInformationMissing')"
+        ></div>
+      </div>
     </div>
   </div>
 </template>
 
 <script setup lang="ts">
 import { productBundleGetters } from '@plentymarkets/shop-sdk';
+import { type ProductBundleComponent } from '@plentymarkets/shop-api';
 import type { BundleOrderItemsProps } from '~/components/BundleOrderItems/types';
 import { SfLink } from '@storefront-ui/vue';
+const { t } = useI18n();
 
 const { product } = withDefaults(defineProps<BundleOrderItemsProps>(), {});
 const NuxtLink = resolveComponent('NuxtLink');
 const localePath = useLocalePath();
+
+function isLinkable(item: ProductBundleComponent): boolean {
+  let returnValue = false;
+  if (productBundleGetters.isItemBundleSalable(item) && !productBundleGetters.getBundleItemUrl(item).includes('null')) {
+    returnValue = true;
+  }
+  return returnValue;
+}
```

Comment:
```suggestion
const isLinkable = (item: ProductBundleComponent): boolean => {
  return productBundleGetters.isItemBundleSalable(item) && !productBundleGetters.getBundleItemUrl(item).includes('null');
}
```
Shorter solution but you have to check lint too.